README.md

```diff
@@ -14,79 +14,93 @@ These components are setup with the install process available in this repository
 
-#### Adding private SSH key to your VM
+#### Adding a private/public key pair to your VM
 
-Add your private ssh key under `~/.ssh/<your_key>.pem`, this is typically the same key that you use to SSH to your host VM, regardless it needs to be a key created on the AWS console so Amazon is aware of it. Then do `chmod 400 ~/.ssh/<your_key>.pem` so your key is not publicly viewable.
+In the VM add your key pair file under `~/.ssh/<your_key_pair>.pem`. This is typically the same key pair that you use to connect to your VM via SSH. This key pair needs to be created on the [AWS console](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) so Amazon is aware of it. Set the privileges of that key pair file to _read-by-user-only_ by `chmod 400 ~/.ssh/<your_key>.pem` so it is not publicly viewable.
 
-#### TODO:
 
-* Guide on choosing AWS instance type... make sure it matches your AMI.
-* AMI, use an ubuntu 16.04 base box, you can use the official Ubuntu release.  You may need to make your own AMI with more storage! Needs to be in your region!  You may want to google to start with the official Ubuntu images for your region.
+## Installing the Platform (CGP)
+### Collecting Information
 
-### Setup for Boardwalk
+The installation script (`install_bootstrap`) will prompt for several questions. To expedite the installation process it is useful to prepare for some of the answers beforehand. Here are a few pointers:
 
-See the Boardwalk [README](boardwalk/README.md) for details.
+* make sure you know what AWS region your VM runs in (e.g. `us-west-2`)
+* decide whether you want to create an instance for development or production as it might impact the size and therefore the cost of the host virtual machine (see above for recommendations)
+* find out whether your favorite editor is installed on the VM
+* create a static IP address for your VM (AWS calls this _Elastic IP_); find a short set of instructions above
+* you will be asked to provide an host domain that points to your EC2 instance; you need to know the name of the domain but at the time of installation the domain (or _record set_) does not have to be configured in _Route 53_
+* _Boardwalk_ has functionality to export metadata to Broad's FireCloud; in order to use it you need to provide your Google Cloud Platform credentials; specifically you need the Google Client ID and the Google Client Secret (it's okay to leave the Google site verfication code empty)
+* all metadata reside in an Elasticsearch database; make sure you have the domain name of that Elasticsearch instance handy
+* have the domain name of the _dos-dss server_ handy
 
 ### Running the Installer
 
-Once the above setup is done, clone this repository onto your server and run the bootstrap script.
+Once the above setup is done, clone this repository onto your server and run the bootstrap script. Be sure to set your branch to `feature/commons` as these instruction are specific to this branch.
```

Comment:
The above is collecting information for the setup, although I suppose you could argue that collecting information is the setup. I'm used to installation programs on Windows being called setup...

I would say `Once you have collected the above information, clone this...`